test/widgets/content_test.dart

```diff
@@ -1028,9 +1062,39 @@ void main() {
       tester.widget(find.text('λ', findRichText: true));
     });
 
-    testWidgets('fallback to displaying KaTeX source if unsupported KaTeX HTML', (tester) async {
-      await prepareContent(tester, plainContent(ContentExample.mathInlineUnknown.html));
-      tester.widget(find.text(r'\lambda'));
+    group('fallback to displaying KaTeX source if unsupported KaTeX HTML', () {
+      testWidgets('smoke', (tester) async {
+        await prepareContent(tester, plainContent(ContentExample.mathInlineUnknown.html));
+        tester.widget(find.text(r'\lambda'));
+      });
+
+      testFontWeight('is bold in bold span',
+        // Regression test for: https://github.com/zulip/zulip-flutter/issues/1812
+        expectedWght: 600,
+        // **bold**
+        content: plainContent('<p><strong><span class="katex">'
+          '<span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>λ</mi></mrow>'
+            '<annotation encoding="application/x-tex"> \\lambda </annotation></semantics></math></span>'
+          '<span class="katex-html" aria-hidden="true">'
+            '<span class="base unknown">' // Server doesn't generate this 'unknown' class.
+            '<span class="strut" style="height:0.6944em;"></span>'
+            '<span class="mord mathnormal">λ</span></span></span></span></strong></p>'),
```

Comment:
How about something like:

```suggestion
        content: plainContent('<p><strong>$unsupportedKatexHtml</strong></p>'),
```

where that string is shared between these two new tests and the existing one above? That way it's clear that the bulk of the logic is in common, and one can more easily focus on the bits that differ.

Reply:
Makes sense; see what you think in my next revision